### ISSUE TITLE
loss function hyperparameters setting

### ISSUE THREAD

**Issue by leowl0924** [albrateanu/LYT-Net#23]
in losses.py,
total_loss = (self.alpha1 * smooth_l1_l + self.alpha2 * perc_l + 
                      self.alpha3 * hist_l + self.alpha5 * psnr_l + 
                      self.alpha6 * color_l + self.alpha4 * ms_ssim_l)
Unlike the parameters in the paper, is there an error here？

**Comment by leowl0924** [comment]
The correct configuration should be？
total_loss = (self.alpha1 * smooth_l1_l + self.alpha2 * perc_l +
self.alpha3 * hist_l + **self.alpha4** * psnr_l +
**self.alpha5** * color_l + **self.alpha6** * ms_ssim_l)

**Comment by albrateanu** [comment]
Hi @leowl0924 

The mistake has occured in the paper rather than the code. The values shown in the code are correct. I will update the paper as soon as possible. 

Thank you.

**Comment by albrateanu** [comment]
The small weight helps it remain in the [0,1] range.
Since the psnr loss computation is 40 - PSNR, and assume PSNR is 25, then loss will be 15. Leaving it as is would be unbalanced with respect to other losses like the Smooth L1, which is also in the [0,1] range. 
Doing 15 * 0.0083 would get you to around 0.12 which gets the thing in the correct interval.

**Comment by leowl0924** [comment]
Thanks for the reply, I'll deploy your model and test.

**Comment by leowl0924** [comment]
Thanks for the reply, I'll deploy your model and test.

**Comment by climberwu** [comment]
Hello,May I ask which journal your paper has been submitted to?

**Comment by leowl0924** [comment]
> Hello,May I ask which journal your paper has been submitted to?

it is albrateanu's paper. you can search it in bing.com
